### PR TITLE
chore: split new rbac resources for chaos-dashboard

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

### What problem does this PR solve?

it's a part of https://github.com/chaos-mesh/chaos-mesh/issues/3015

### What's changed and how it works?

- new file `chaos-dashboard-rbac.yaml` includes related `ServiceAccount`, `Role`, `ClusterRole`, `RoleBinding` and `ClusterRoleBinding` for chaos-dashboard
- update related `install.sh`

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [x] Manual test (add steps below)

install Chaos Mesh at my 

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Comment by STRRL** [comment]
PTAL

/cc @YangKeao @iguoyr @g1eny0ung @WangXiangUSTC @cwen0

File: helm/chaos-mesh/templates/chaos-dashboard-rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# Copyright 2021 Chaos Mesh Authors.
```
Comment (g1eny0ung) [inline helm/chaos-mesh/templates/chaos-dashboard-rbac.yaml:1]: 2022 😆.

**Review by iguoyr — APPROVED** [review]
LGTM

File: helm/chaos-mesh/templates/chaos-dashboard-rbac.yaml (hunk 1 above)
Comment (STRRL) [reply helm/chaos-mesh/templates/chaos-dashboard-rbac.yaml]: updated PTAL!

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Pull request merged**